docs/bugs.json

```diff
@@ -1,14 +1,22 @@
 [
     {
+        "name": "DelegateCallReturnValue",
+        "summary": "The low-level .delegatecall() takes the returned value as a boolean instead of the execution outcome.",
+        "description": "If the destination returns a value or garbage, which if interpreted as a boolean means false, the return value of .delegatecall() is false, despite successful execution.",
```

Comment:
This happens even if the destination does not return anything (not even garbage).

What about: `The return value of the low-level .delegatecall() function is taken from a position in memory, where the call data or the return data resides. This value is interpreted as a boolean and put onto the stack. This means if the called function returns at least 32 zero bytes, .delegatecall() returns false even if the call was successuful.'